source/clear-linux/how-to/swupd/run.rst

```diff
@@ -1,59 +1,60 @@
 .. _swupd-run:
 
-How to run the updater
-######################
+How to update Clear Linux
+#########################
 
-This section describes how to manually trigger the software update code in
-Clear Linux\* OS for Intel® Architecture.
+This section describes how to update |CLOSIA|.
 
 Prerequisites
 =============
 
--  System is up and running a Clear Linux OS for Intel
-   Architecture official build, 7420 later.
+* The system is up and running the Clear Linux 15400 build or later.
 
--  Device is on a well-connected network.
+* Device is on a well-connected network.
 
--  Device is able to connect to release infrastructure. For example:
-   http://update.clearlinux.org/update/300/
+* Device is able to connect to the release infrastructure. For example:
```

Comment:
Agree.  Let's change the last 2 lines to simply "Device is able to connect to the release infrastructure http://update.clearlinux.org/.